src/kanata/mod.rs

```diff
@@ -466,6 +466,11 @@ impl Kanata {
                 let ret = self.handle_repeat(event);
                 return ret;
             }
+            KeyValue::Tap => {
```

Comment:
Optional: this could be avoided by having the event loop itself send the two events over the channel instead of a single event. It seems fine as-is though.

Reply:
Actually it seems that this concept should be quite helpful to the Interception code, so LGTM